dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -434,6 +436,22 @@ def valid_s3_bucket_name(name: str):
 
         return True
 
+    @staticmethod
+    def __datadog_schema() -> dict:
+        return {
+            "type": "datadog",
+            Optional("environments"): {
+                PlatformConfigSchema.__valid_environment_name(): {
+                    Optional("team_name"): str,
+                    Optional("contact_name"): str,
+                    Optional("contact_email"): str,
+                    Optional("repository"): str,
```

Comment:
Might be worth being clear that this is referring to the codebase repository and that it's a URL, not just `uktrade/<application>`?
```suggestion
                    Optional("codebase_repository_url"): str,
```

Reply:
This was removed.